scss/civicrm/advanced-search/pages/_results.scss

```diff
@@ -1,5 +1,6 @@
 .CRM_Contact_Form_Search_Advanced .crm-results-block,
-.CRM_Contact_Form_Search_Basic .crm-results-block {
+.CRM_Contact_Form_Search_Basic .crm-results-block,
```

Comment:
the folder containing this partial is "advanced-search", but here there are other type of search pages mentioned in the selectors. Can't we find a better placement for this?